README.md

```diff
@@ -158,9 +166,599 @@ search.start();
+
+The `breadcrumb` widget is a secondary navigation scheme that lets the user see where the current page is in relation to the facet’s hierarchy.
+
+No compatibility because MeiliSearch does not support hierarchical facets.
+
+If you'd like get nested facets implemented, please vote for it in the [roadmap](https://roadmap.meilisearch.com/c/97-nested-facets?utm_medium=social&utm_source=portal_share).
+
+### ✅ Stats
+
+[Stats references](https://www.algolia.com/doc/api-reference/widgets/stats/js/)
+
+The `stats` widget displays the total number of matching hits and the time it took to get them (time spent in the Algolia server).
+
+- ✅ container: The CSS Selector or HTMLElement to insert the widget into. _required_
+- ✅ cssClasses: The CSS classes to override.
+- ✅ transformItems: function receiving the items, called before displaying them.
+
+```js
+instantsearch.widgets.stats({
+  container: '#stats',
+})
+```
+
+### ❌ Analytics
+
+[Analytics](https://www.algolia.com/doc/api-reference/widgets/analytics/js/)
+
+Deprecated. See [Insight](#Insight).
+
+### ❌ QueryRuleCustomData
+
+[QueryRuleCustomData references](https://www.algolia.com/doc/api-reference/widgets/query-rule-custom-data/js/)
+
+You may want to use this widget to display banners or recommendations returned by Rules, and that match search parameters.
+
+No compatibility because MeiliSearch does not support [rules](https://www.algolia.com/doc/guides/managing-results/rules/rules-overview/).
+
+### ❌ QueryRuleContext
+
+[Query rule context references](https://www.algolia.com/doc/api-reference/widgets/query-rule-context/js/)
+
+The queryRuleContext widget lets you apply ruleContexts based on filters to trigger context-dependent Rules.
+
+No compatibility because MeiliSearch does not support [rules](https://www.algolia.com/doc/guides/managing-results/rules/rules-overview/).
+
+### ❌ SortBy
+
+[Sort by references](https://www.algolia.com/doc/api-reference/widgets/sort-by/js/)
+
+The sortBy widget displays a list of indices, allowing a user to change the way hits are sorted (with replica indices). Another common use case is to let the user switch between different indices.
+
+No compatibility because MeiliSearch does not support hierarchical facets.
+
+If you'd like to get the sort by features, please vote for it in the [roadmap]https://roadmap.meilisearch.com/c/32-sort-by?utm_medium=social&utm_source=portal_share).
+
+### ❌ RelevantSort
+
+[Relevant Sort references](https://www.algolia.com/doc/api-reference/widgets/relevant-sort/js/)
+
+Virtual indices allow you to use Relevant sort, a sorting mechanism that favors relevancy over the attribute you’re sorting on.
+
+### ✅ Routing
+
+Routing is configured inside `instantSearch` component. Please refer [to the documentation](https://www.algolia.com/doc/api-reference/widgets/simple-state-mapping/js/) for further implementation information.
+
+## 🤖 Compatibility with MeiliSearch and InstantSearch
```

Comment:
Can you out this section before ``. I'm worried the users never reach this section since the README is huge.

Reply:
Sorry, I wanted to say the section before "Example with InstantSearch"